Libraries/Lists/FlatList.js

```diff
@@ -614,24 +614,72 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
           return (
             <View style={StyleSheet.compose(styles.row, columnWrapperStyle)}>
               {item.map((it, kk) => {
-                const element = renderer({
-                  item: it,
-                  index: index * numColumns + kk,
-                  separators: info.separators,
-                });
+                const accessibilityCollectionItem = {
+                  rowIndex: index,
+                  rowSpan: 1,
+                  columnIndex: (index * numColumns + kk) % numColumns,
+                  columnSpan: 1,
+                  heading: false,
+                  itemIndex: index * numColumns + kk,
```

Comment:
Nit: now that `index` is used twice, it would be nice to declare it as a const ex `const itemIndex = index * numColumns + kk` just to avoid errors where someone changes the calculation on one line but not the other.